src/components/molecules/EarthEditor/PropertyPane/PropertyField/ColorField/hooks.ts

```diff
@@ -0,0 +1,149 @@
+import { useState, useEffect, useCallback, useRef } from "react";
+import tinycolor, { ColorInput } from "tinycolor2";
+
+export type RGBA = {
+  r: number;
+  g: number;
+  b: number;
+  a: number;
+};
+
+type Params = {
```

Comment:
Export it because it is used by an exported function

Reply:
Ok!